"use server";

export const signIn = async () => {
  try {
  } catch (error) {
    console.log("Error", error);
  }
};

export const signUp = async () => {
  try {
  } catch (error) {
    console.error("Error", error);
  }
};
